### ISSUE TITLE
Clarify the documentation for the Spinodal Model (Trac #1167)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#1206]
Xin Xu at Imperial has raised some questions about the interpretation and parameterisation of the spinodal model.

Migrated from http://trac.sasview.org/ticket/1167
```json
{
    "status": "closed",
    "changetime": "2018-09-07T15:49:26",
    "_ts": "2018-09-07 15:49:26.273337+00:00",
    "description": "Xin Xu at Imperial has raised some questions about the interpretation and parameterisation of the spinodal model.",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2018-09-05T12:26:24",
    "component": "SasView",
    "summary": "Clarify the documentation for the Spinodal Model",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "smk78",
    "type": "task"
}
```

**Comment by smk78** [comment]
Trac update at `2018/09/05 12:27:13`: **smk78** changed summary from "Clarity the documentation for the Spinodal Model" to "Clarify the documentation for the Spinodal Model"

**Comment by smk78** [comment]
Trac update at `2018/09/07 15:49:26`: 
* **smk78** commented:

In changeset 475ff585df3808f7c560003331e4cf2e1ea87fc3:

```text
#!CommitTicketReference repository="sasmodels" revision="475ff585df3808f7c560003331e4cf2e1ea87fc3"
Clarified documentation for spinodal model. Closes #1206
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"
